pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -123,6 +123,7 @@ func BuildKubeAPIserverOptions(masterConfig configapi.MasterConfig) (*kapiserver
 	server.SecureServing.ServerCert.CertKey.KeyFile = masterConfig.ServingInfo.ServerCert.KeyFile
 	server.InsecureServing.BindPort = 0
 
+	server.Authentication.Anonymous.Allow = false
```

Comment:
We don't use the authenticator or authorizer created by this anywhere, do we?  If we don't, write a scary comment and this is ok for the rebase.